provider/cmd/pulumi-resource-hcloud/schema.json

```diff
@@ -3367,7 +3367,7 @@
l {\n\t\t\treturn err\n\t\t}\n\t\t// Create a new server from the snapshot\n\t\t_, err = hcloud.NewServer(ctx, \"from_snapshot\", \u0026hcloud.ServerArgs{\n\t\t\tName:       pulumi.String(\"from-snapshot\"),\n\t\t\tImage:      pulumi.Int(packerSnapshot.Id),\n\t\t\tServerType: pulumi.String(\"cx11\"),\n\t\t\tPublicNets: hcloud.ServerPublicNetArray{\n\t\t\t\t\u0026hcloud.ServerPublicNetArgs{\n\t\t\t\t\tIpv4Enabled: pulumi.Bool(true),\n\t\t\t\t\tIpv6Enabled: pulumi.Bool(true),\n\t\t\t\t},\n\t\t\t},\n\t\t})\n\t\tif err != nil {\n\t\t\treturn err\n\t\t}\n\t\treturn nil\n\t})\n}\n```\n```java\npackage generated_program;\n\nimport com.pulumi.Context;\nimport com.pulumi.Pulumi;\nimport com.pulumi.core.Output;\nimport com.pulumi.hcloud.HcloudFunctions;\nimport com.pulumi.hcloud.inputs.GetImageArgs;\nimport com.pulumi.hcloud.Server;\nimport com.pulumi.hcloud.ServerArgs;\nimport com.pulumi.hcloud.inputs.ServerPublicNetArgs;\nimport java.util.List;\nimport java.util.ArrayList;\nimport java.util.Map;\nimport java.io.File;\nimport java.nio.file.Files;\nimport java.nio.file.Paths;\n\npublic class App {\n    public static void main(String[] args) {\n        Pulumi.run(App::stack);\n    }\n\n    public static void stack(Context ctx) {\n        // Get image infos because we need the ID\n        final var packerSnapshot = HcloudFunctions.getImage(GetImageArgs.builder()\n            .withSelector(\"app=foobar\")\n            .mostRecent(true)\n            .build());\n\n        // Create a new server from the snapshot\n        var fromSnapshot = new Server(\"fromSnapshot\", ServerArgs.builder()        \n            .name(\"from-snapshot\")\n            .image(packerSnapshot.applyValue(getImageResult -\u003e getImageResult.id()))\n            .serverType(\"cx11\")\n            .publicNets(ServerPublicNetArgs.builder()\n                .ipv4Enabled(true)\n                .ipv6Enabled(true)\n                .build())\n            .build());\n\n    }\n}\n```\n```yaml\nresources:\n  # Create a new server from the snapshot\n  fromSnapshot:\n    type: hcloud:Server\n    name: from_snapshot\n    properties:\n      name: from-snapshot\n      image: ${packerSnapshot.id}\n      serverType: cx11\n      publicNets:\n        - ipv4Enabled: true\n          ipv6Enabled: true\nvariables:\n  # Get image infos because we need the ID\n  packerSnapshot:\n    fn::invoke:\n      Function: hcloud:getImage\n      Arguments:\n        withSelector: app=foobar\n        mostRecent: true\n```\n\u003c!--End PulumiCodeChooser --\u003e\n\n## Primary IPs\n\nWhen creating a server without linking at least one ´primary_ip´, it automatically creates \u0026 assigns two (ipv4 \u0026 ipv6).\nWith the public_net block, you can enable or link primary ips. If you don't define this block, two primary ips (ipv4, ipv6) will be created and assigned to the server automatically.\n\n## Import\n\nServers can be imported using the server `id`:\n\n```sh\n$ pulumi import hcloud:index/server:Server myserver id\n```\n\n",
```

Comment:
Seems like some of the examples got removed here? Why's that?

Reply:
This is back now.